static/js/containers/pages/admin/DeactivateCouponPage_test.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import { assert } from "chai";
+import * as sinon from "sinon";
+
+import DeactivateCouponPage, {
+  DeactivateCouponPage as InnerDeactivateCouponPage,
+} from "./DeactivateCouponPage";
+
+import IntegrationTestHelper from "../../../util/integration_test_helper";
+import { Modal } from "reactstrap";
+import wait from "waait";
+
+describe("DeactivateCouponPage", () => {
+  let helper, renderDeactivateCouponPage, setSubmittingStub, setErrorsStub;
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper();
+    setSubmittingStub = helper.sandbox.stub();
+    setErrorsStub = helper.sandbox.stub();
```

Comment:
Are we using this? What is the usage of setErrors?

Reply:
No, we are not using this. Removing in the next commit